examples/example-template.sql

```diff
@@ -0,0 +1,115 @@
+-- ===============================================================
+-- Description:
+-- Here you describe what the example does.
+-- Your description should follow the following format:
+-- In this query, <description>.
+-- ===============================================================
+
+-- Variable creation:
+-- Describe what variables are being defined.
+-- For Luminescence examples, 'luminescence-example'
+-- should be used for the PortfolioScope.
+
+-- Example:
+-- Defining portfolio scope and code.
+@@portfolioScope =
+
```

Comment:
Put all on one line?
You might have copied this from another example which is incorrect. 


```
@@portfolioScope =cselect 'luminesce-examples';
```